src/blog/features-from-version-5.md

```diff
@@ -0,0 +1,110 @@
+```javascript
+module.exports = {
+  compilers: {
+    solc: {
+      version: “pragma”
+    }
+  },
+  // … the rest of your config goes here
+};
+```
+Now you can run `truffle compile` and your project will compile!
+
+In `v5.2-example-box` referenced above, you can see how the `truffle-config.js` is set up to use this new experimental feature. Notice that `contracts/MetaCoin.sol` requires `^0.7.0` and how `contracts/Conversion.sol` requires `^0.6.0`. When running `truffle compile` on this example project you should get a printout of all the versions of the Solidity compiler used during compilation. Namely, you should see that both version `0.6.12` and `0.7.6` are listed.
+
+As mentioned above, when you use this feature, Truffle will analyze each or your Solidity sources one by one to find the appropriate version of the Solidity compiler for each source. Do note, however, that each source file and all of its imports must be compiled with one version of Solidity; this means there must be a version of the compiler that satisfies all of their pragma expressions.
+
+## Things you may have missed in Truffle 5.1
+
+We’ve added a fair bit over the course of Truffle 5.1 as well, and there’s a good chance you didn’t catch all of it!  We thought this would be a good time to review some of that as well!
+
+### Compilation of abi.json files
+
+Another exciting addition to Truffle is the ability to compile from a contract’s ABI. When you create a JSON source file in your “contracts” directory that contains a contract’s ABI, Truffle will take it, use it to create an interface, and compile it. This might come in handy, for example, when you want to import and use a library that requires a different version of the Solidity compiler than the importing file. You can then simply compile the library and create a new JSON file containing the ABI of the imported library. Under the hood Truffle will compile the ABI into a Solidity interface and create an artifact for it. You will then be able to import the interface into your contract!
+
+In the v5.2-example-box, consider the `contracts/MetaCoin.sol` and `contracts/Conversion.sol`. We would like to import Conversion into the MetaCoin contract to use it but cannot since they require different versions of the Solidity compiler.  To work around this, we have created `contracts/IConversion.abi.json` which contains the ABI from the compiled Conversion. In this way we can now import the interface into `contracts/MetaCoin.sol` (see line 4 in the MetaCoin contract) to use it!
+
+### Solidity stacktraces in Truffle Test
+
+Try running your tests with the `--stacktrace` option (or `-t`) for short and get combined Solidity-Javascript stacktraces!  This feature does have some limitations and is still somewhat experimental, but we expect you’ll find it quite useful.  In the future we may add support for Vyper stacktraces as well!  
```

Comment:
probably should be "option (or `-t` for short)"

Reply:
Oops, good catch!